README.md

```
@@ -1,26 +1,26 @@
 
 ## Setup
 
 1. Copy `.env.example` to `.env` and fill in your Shopify tokens.
-2. Install dependencies with `yarn install` or `npm install`.
-3. Start the dev server with `yarn start`.
+2. Install dependencies with `yarn install`.
+3. Start the dev server with `yarn dev`.
 
 ### Required environment variables
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)